tests/benchdnn/conv/conv.cpp

```diff
@@ -667,6 +667,32 @@ void check_known_skipped_case(const prb_t *prb, res_t *res) {
     if (prb->alg == WINO) {
         if (is_cpu()) {
 #if DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+#if DNNL_AARCH64
+            bool pad_ok_f32 = prb->pw <= 1 && prb->ph <= 1 && prb->pw_r <= 1
+                    && prb->ph_r <= 1;
+            bool shape_ok = prb->ndims == 4 && prb->g == 1 && prb->kh == 3
+                    && prb->kw == 3 && prb->sh == 1 && prb->sw == 1
+                    && prb->dh == 0 && prb->dw == 0 && !prb->has_groups
+                    && pad_ok_f32;
+
+            const auto stag = normalize_tag(prb->stag, prb->ndims);
+            const bool stag_is_valid = stag == "any";
+            const auto dtag = normalize_tag(prb->dtag, prb->ndims);
+            const bool dtag_is_valid = dtag == "any";
+            const bool format_ok = (stag_is_valid || dtag_is_valid);
```

Comment:
Could you, please, clarify this condition more: what do you want to achieve with it?

Reply:
We wanted to check the tags were nhwc, but there doesn't appear to be an option, so I added this check here to, at the very least, check the tags aren't undef.

Reply:
I believe the library would return a proper `invalid_arguments` status in case of `tag::undef`. This tag doesn't sound right for convolution, thus, I don't think such intention should be put there...
In case you don't utilize any input files but CI or your own, and as long as there's no UNIMPLEMENTED failing testing runs, I think it's fun to drop auxiliary conditions.

We have them here since many configurations show up in different input files and we don't want to figure out every time whether this is an expected failure or not.

Reply:
Okay, I've removed this check.